server/jwt.go

```
@@ -0,0 +1,79 @@
+}
+
+// Purpose of this cookie is to store the user's id.
+func setUserSession(c echo.Context, user *api.User) {
+	sess, _ := session.Get("session", c)
+	sess.Options = &sessions.Options{
+		Path:     "/",
+		MaxAge:   1000 * 3600 * 24 * 30,
+		HttpOnly: true,
+	}
+	sess.Values[userIdContextKey] = strconv.Itoa(user.Id)
+	sess.Save(c.Request(), c.Response())
```

golangci-lint (1.64.8)

Error return value of `sess.Save` is not checked

(errcheck)